src/views/catalog/CreateFromInstanceTypes/state/hooks/useInstanceTypesAndPreferences.ts

```diff
@@ -1,23 +1,28 @@
+import useUserInstanceTypes from '@catalog/CreateFromInstanceTypes/state/hooks/useUserInstanceTypes';
+
 import { UseInstanceTypeAndPreferencesValues } from '../utils/types';
 
+import useClusterInstanceTypes from './useClusterInstanceTypes';
 import useClusterPreferences from './useClusterPreferences';
-import useInstanceTypes from './useInstanceTypes';
 
 type UseInstanceTypeAndPreferences = () => UseInstanceTypeAndPreferencesValues;
 
 const useInstanceTypesAndPreferences: UseInstanceTypeAndPreferences = () => {
-  const [instanceTypes, instanceTypesLoaded, instanceTypesLoadError] = useInstanceTypes();
+  const [clusterInstanceTypes, clusterITsLoaded, clusterITsLoadError] = useClusterInstanceTypes();
+
+  const [userInstanceTypes, userITsLoaded, userITsLoadError] = useUserInstanceTypes();
 
   const [preferences, preferencesLoaded, preferencesLoadError] = useClusterPreferences();
 
-  const loaded = preferencesLoaded && instanceTypesLoaded;
-  const loadError = preferencesLoadError || instanceTypesLoadError;
+  const loaded = preferencesLoaded && clusterITsLoaded;
+  const loadError = preferencesLoadError || clusterITsLoadError;
 
   return {
-    instanceTypes,
-    loaded: loaded || !!loadError,
+    clusterInstanceTypes,
+    loaded: (loaded && userITsLoaded) || !!loadError || !!userITsLoadError,
```

Comment:
u can add the values for line 17, 18

Reply:
Done! I definitely wasn't thinking clearly there. Thanks for catching it!